onnxruntime/test/providers/nv_tensorrt_rtx/nv_basic_test.cc

```diff
@@ -22,6 +23,21 @@ namespace onnxruntime {
 
 namespace test {
 
+// Helper function to check if GPU is Blackwell (SM 12.0+) or above
+// Returns true if requirement is met
+// Returns false if CUDA is unavailable or GPU is below SM 12.0
+static bool IsBlackwellOrAbove() {
+  constexpr int kBlackwellMinCapability = 1200;  // SM 12.0 = 12 * 100 + 0 * 10
```

Comment:
1000?  see https://developer.nvidia.com/cuda/gpus. If we want to run those tests in B200 or B300 

It's probably fine if rtx is targeted for consumer GPUs only.